src/pretix/presale/context.py

```diff
@@ -53,6 +54,8 @@ def contextprocessor(request):
             ctx['css_file'] = default_storage.url(request.event.settings.presale_css_file)
         ctx['event_logo'] = request.event.settings.get('logo_image', as_type=str, default='')[7:]
         ctx['event'] = request.event
+        for code in request.event.settings.locales:
+            ctx.setdefault('languages',[]).append( get_language_info(code) )
```

Comment:
Hmm, I'm not really a fan of calling ``setdefault`` that many times, although it probably doesn't matter. I think a list comprehension would be cleaner:
```
ctx['languages'] = [
    get_language_info(code) for code in request.event.settings.locales
]
```
What do you think?